packages/orchestrator/claim_schema.py

```
@@ -0,0 +1,221 @@
+
+    claim_id: str
+    outcome: Outcome
+    votes: list[Vote]
+    decided_at: str = field(default_factory=_utcnow_iso)
+    adr_ref: Optional[str] = None
+    override_by: Optional[str] = None
+
+    def validate(self) -> None:
+        """Enforce spec invariant INV-009."""
+        if self.outcome == Outcome.OVERRIDDEN and not self.override_by:
+            raise ValueError("E_OVERRIDE_NOT_HUMAN: OVERRIDDEN requires override_by")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)